test/functional/sync_plan/list_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+
+module HammerCLIKatello
+  describe SyncPlan::ListCommand do
+    it 'allows minimal options' do
+      api_expects(:sync_plans, :index) { |p| p['organization_id'] == 1 }
+
+      run_cmd(%w(sync-plan list --organization-id 1))
+    end
+
```

Comment:
I would add a describe block here for organization options, surrounding the 'name' and 'label' tests:

```rb
describe 'organization ID can be resolved from organization'
...
end
```